apptools/preferences/ui/i_preferences_page.py

```diff
@@ -21,12 +21,3 @@ class IPreferencesPage(Interface):
 
     def apply(self):
         """ Apply the page's preferences. """
-
```

Comment:
Part of me would want to see `pass` here (which is probably the more consistent thing to do?). I know I did not do that on another occasion when I removed the bare returns due to laziness. Just wanted to make sure I have not accidentally advocated a pattern when I did not mean it 😓 , sorry about that.